guppy/src/graph/graph.rs

```diff
@@ -350,6 +350,13 @@ impl PackageGraph {
     }
 }
 
+impl Default for PackageGraph {
+    fn default() -> Self {
+        let mut command = MetadataCommand::new();
+        Self::from_command(&mut command).expect("default metadata command failed")
+    }
+}
```

Comment:
To be honest this is too magical -- I would not expect a `Default` impl to execute a subprocess. This doesn't really appear hugely necessary -- maybe just inline this code in its one caller?

Reply:
That's fair. Removed.